tests/rptest/perf/small_batches_test.py

```diff
@@ -30,10 +30,20 @@ def omb_test(self):
             "test_duration_minutes": 5,
             "warmup_duration_minutes": 1,
         }
+        validator = {
+            OMBSampleConfigurations.E2E_LATENCY_50PCT:
+            [OMBSampleConfigurations.lte(30)],
+            OMBSampleConfigurations.E2E_LATENCY_AVG:
+            [OMBSampleConfigurations.lte(75)],
+            OMBSampleConfigurations.AVG_THROUGHPUT_MBPS:
+            [OMBSampleConfigurations.gte(145)],
+            OMBSampleConfigurations.PUB_LATENCY_50PCT:
+            [OMBSampleConfigurations.lte(40)],
+        }
 
-        benchmark = OpenMessagingBenchmark(
-            self._ctx, self.redpanda, "ACK_ALL_GROUP_LINGER_1MS",
-            (workload, OMBSampleConfigurations.UNIT_TEST_LATENCY_VALIDATOR))
+        benchmark = OpenMessagingBenchmark(self._ctx, self.redpanda,
+                                           "ACK_ALL_GROUP_LINGER_1MS",
```

Comment:
Did we ever check what effective batch size this results in? This seems more like a many partition test than a small batch size one. I'd also be surprised if the client machines can handle 200 producers with 20k partitions. I expect we get lots of client induced latency here.